CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog - v3
 
+## [v3.4.4] (Mar 31 2023)
+Features:
+* Increase the default maximum recording time of Voice Message to 10-minute
+* Add logger to VoicePlayer, VoiceRecorder, and useSendVoiceMessage hook
+
+Fixes:
+* Do not allow to scroll the whole page when OpenChannel is inside of the page that has scroll
+* Fix the issue that sending two voice messages when play audio before send the message
```

Comment:
minor; (don't need to apply it)
How about this? I honestly had to read 2-3 times to get this ㅋㅋ 
```suggestion
* Fixed the issue of sending two voice messages when playing audio before sending the message.
```